tests/unit/auth.spec.ts

```diff
@@ -14,7 +14,7 @@ const openIDConfig = {
 describe('auth.ts', () => {
   beforeEach(async () => {
     mockAxios.get.mockResolvedValueOnce(openIDConfig);
-    await keycloakConfiguration.startConfiguration();
+    keycloakConfiguration.startConfiguration();
     expect(mockAxios.get).toHaveBeenCalledWith(config.auth.keycloak.configurationURL);
   });
```

Comment:
actually the retrieval should be synchronous to have a deterministic result, but it is not with and without the `await`. I'm not sure if the test should rather be deleted and a new one should be created in the issue for the tests (Gamify-IT/issues#257)